docs/references/go/other-examples.mdx

```diff
@@ -5,30 +5,77 @@ description: Explore different examples of how to utilize Clerk with Go.
 
 # Examples
 
-Below is a code snippet showing what you can do from your backend with Clerk. Look at the [API reference](/docs/references/backend/overview) for a full list of requests you can make from your backend.
+The following example demonstrates how to use the Clerk Go SDK to execute [Clerk Backend API](/docs/references/backend/overview) operations.
 
-## Update metadata
+By executing the code in the snippet below, you will:
+- Create an organization and update its slug.
+- Fetch all organization memberships and loop through them to get the first one.
+- Get more details about the organization's user.
 
-Add a new key/value to the private metadata stored on a user.
+<Callout type="info">
+Your Clerk secret key is required. If you are signed into your Clerk Dashboard, your secret key should become visible by clicking on the eye icon. Otherwise, you can retrieve your Clerk secret key from the Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
```

Comment:
```suggestion
Your Clerk secret key is required. If you are signed into your Clerk Dashboard, your secret key should become visible by selecting the eye icon. Otherwise, you can retrieve your Clerk secret key from the Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
```

selecting = more accessible language

Reply:
I agree! I've been slowly integrating it into the docs - definitely missed that O: